ee/clickhouse/sql/person.py

```diff
@@ -320,3 +299,22 @@
 GROUP BY tupleElement(keysAndValues, 1)
 ORDER BY count DESC, key ASC
 """
+
+GET_PERSONS_FROM_EVENT_QUERY = """
+SELECT
+    person_id,
+    created_at,
+    team_id,
+    person_props,
+    is_identified,
+    arrayReduce('groupUniqArray', groupArray(distinct_id)) AS distinct_ids
```

Comment:
still worried this could be a mess with some of person_ids with uniquely large amount of distinct_ids 🤷‍♂️

Reply:
Agreed - we should look into making person links person_id based at some point which would solve this mess. We _could_ also just do `[any(distinct_id)] AS distinct_ids` but :man_shrugging: